pkg/cri/streaming/server.go

```diff
@@ -162,9 +162,10 @@ func NewServer(config Config, runtime Runtime) (Server, error) {
 	handler.Add(ws)
 	s.handler = handler
 	s.server = &http.Server{
-		Addr:      s.config.Addr,
-		Handler:   s.handler,
-		TLSConfig: s.config.TLSConfig,
+		Addr:              s.config.Addr,
+		Handler:           s.handler,
+		TLSConfig:         s.config.TLSConfig,
+		ReadHeaderTimeout: 3 * time.Second, // Fix linter G112: Potential Slowloris Attack because ReadHeaderTimeout is not configured in the http.Server
```

Comment:
No sure 3 second is good enough. Just in case, the 10 Minutes maybe better than 3 second because there is no limit at the beginning. 

It's not block issue. But if we change it to low tolerance value, we should mark it in change log.

Cc @mikebrow

Reply:
yeah this one is very likely to cause node failures..

Reply:
I agree maybe 10 or 15min.. maybe 30?